gax-java/gax-grpc/src/main/resources/META-INF/native-image/com.google.api/gax-grpc/native-image.properties

```diff
@@ -13,4 +13,4 @@ Args=--add-opens=java.base/java.time=ALL-UNNAMED \
     io.grpc.netty.shaded.io.netty \
   --features=com.google.api.gax.grpc.nativeimage.ProtobufMessageFeature,\
   com.google.api.gax.grpc.nativeimage.GrpcNettyFeature \
-  -H:-RunReachabilityHandlersConcurrently
\ No newline at end of file
+  -H:-RunReachabilityHandlersConcurrently
```

Comment:
`ProtobufMessageFeature` and `-H:-RunReachabilityHandlersConcurrently` have now been reintroduced.

Implementation plan:
1. Merge and release with this PR's changes. 
2. Wait for Owlbot to use the updated gapic-generator-java, and generate the reflect-config.json for each client libary.
3. In the next release cycle: Remove `ProtobufMessageFeature` and `-H:-RunReachabilityHandlersConcurrently`, perform downstream testing on the client libraries that will then have local reflect-configs.